apps/artwork/components/commercial/test/template.coffee

```diff
@@ -16,7 +16,8 @@ render = (templateName) ->
   )
 
 renderArtwork = (artworkOptions = {}, sdOptions = {}) ->
-  artwork = _.extend(inquireableArtwork.data.artwork, artworkOptions)
+  artwork = _.clone inquireableArtwork.data.artwork
```

Comment:
I'm also a bit fuzzy on how stuff gets included/required, but [extend](http://underscorejs.org/#extend) _will_ modify the destination object.